src/RawPOSIX/src/safeposix/dispatcher.rs

```diff
@@ -203,79 +203,197 @@ pub fn lind_syscall_api(
 
     let ret = match call_number {
         WRITE_SYSCALL => {
+            // NaCl equivalent: NaClSysWrite
```

Comment:
Lets get rid of all of these "NaCl equivalent comments" Its good that you checked all of these but we don't need to reference nacl in our new code.

Reply:
Okie